packages/reactstrap-validation-select/resources.js

```diff
@@ -47,6 +50,153 @@ const AvCodeSelect = AvResourceSelect.create({
   watchParams: ['list'],
 });
 
+/* eslint-disable react/prop-types */
+const PatientOption = props => {
```

Comment:
Can we move this to another file? Just to the `resources.js` file clean.

Reply:
yeah good call. i'll update

Reply:
7058a9931b8c459c0efacc28872e087864893352